### ISSUE TITLE
A few common English words that the API reported as non-words during my recent use

### ISSUE THREAD

**Issue by bjkeefe** [meetDeveloper/freeDictionaryAPI#141]
their, every, lives, phase, touch, tight, fewer, doses, flaps, phish

I verified each of these just now by typing them into the "Try it here" box.

No biggie, for what I was working on, but I thought you might like to know.

Thanks again for providing this service.

**Comment by jbalcorn** [comment]
Just found that 'THEIR' isn't matching

**Comment by jbalcorn** [comment]
I just checked the english word list is 'their' is listed, so...something's wrong.  Here's my call:

S C:\users\jalcorn> Invoke-WebRequest -Method GET -Uri "https://api.dictionaryapi.dev/api/v2/entries/en/their"
Invoke-WebRequest : {"title":"No Definitions Found","message":"Sorry pal, we couldn't find definitions for the word you were looking for.","resolution":"You can try the search again at later time or head to the web instead."}
At line:1 char:1
+ Invoke-WebRequest -Method GET -Uri "https://api.dictionaryapi.dev/api ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (System.Net.HttpWebRequest:HttpWebRequest) [Invoke-WebRequest], WebException
    + FullyQualifiedErrorId : WebCmdletWebResponseException,Microsoft.PowerShell.Commands.InvokeWebRequestCommand

**Comment by bjkeefe** [comment]
@jbalcorn Yep,  That's pretty much what I see for all of the words I listed above.
